frontend/components/validator/charts/CumulativeExtrinsicsChart.vue

```diff
@@ -0,0 +1,585 @@
+<template>
```

Comment:
Rename to `CumulativeSignedExtrinsicsChart`

Reply:
How does it relate to `validator`? Move it to the right folder.

Reply:
Moved to charts folder inside components